### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Richienb**
Since the master branch is protected, I'm opening a pull request here.

___

Fixes: #11, Fixes: #12, Fixes: #15

// @stephenmathieson

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,11 +1,9 @@
+const tlds = require('.');
+const test = require('ava');
```
Comment (stephenmathieson) [inline test.js:2]: There is no need for a testing framework here. This is hugely over-engineered. What's wrong with the previous `test.js` file?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = tab
```
Comment (stephenmathieson) [inline .editorconfig:10]: I prefer using spaces for indentation. What is the motivation behind this change?

**Comment by Richienb** [comment]
@stephenmathieson Accustomed to the contribution techniques implied by Sindre Sorhus, I subconsciously made some changes that don't look to be accepted well here.

> I'm not familiar with xo. What benefit is it providing here?

`xo` is an eslint wrapper with excellent defaults.

> I don't think this is a helpful thing to say.

*You should see the other guy*

https://github.com/sindresorhus/video-extensions#readme

> I prefer using spaces for indentation. What is the motivation behind this change?

I recommend you check out https://lea.verou.me/2012/01/why-tabs-are-clearly-superior. However, if tabs still don't float your boat, I'll revert it to 2 spaces.

> There is no need for a testing framework here. This is hugely over-engineered. What's wrong with the previous test.js file?

https://github.com/sindresorhus/video-extensions/blob/master/test.js

> Neither main nor bin entries need to be included here.

I like being explicit with these fields.

File: .github/workflows/cron.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm test
+
+  publish-npm:
+    needs: build
```
[the remark this answers is not in the document]
Comment (Richienb) [reply .github/workflows/cron.yml:19]: 👍

File: .github/workflows/cron.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
```
[the remark this answers is not in the document]
Comment (Richienb) [reply .github/workflows/cron.yml:26]: 👍

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```
[the remark this answers is not in the document]
Comment (Richienb) [reply .github/workflows/main.yml:15]: 👍

**Comment by stephenmathieson** [comment]
> @stephenmathieson Accustomed to the contribution techniques implied by Sindre Sorhus, I subconsciously made some changes that don't look to be accepted well here.

I won't necessarily reject the changes. I'm just a bit overwhelmed by the number of changes you're making in a PR entitled "_Update cron job_".

> xo is an eslint wrapper with excellent defaults.

ESLint has great defaults on its own. The "recommended" ruleset is usually enough IMO.

> You should see the other guy [...] https://github.com/sindresorhus/video-extensions#readme

Yeah, that's not helpful either.

> I recommend you check out https://lea.verou.me/2012/01/why-tabs-are-clearly-superior. However, if tabs still don't float your boat, I'll revert it to 2 spaces.

I have zero interest in partaking in an argument about spaces/tabs. I just don't understand why you opted to change it is all 🤷 

> I like being explicit with these fields.

OK, works for me.

**Comment by stephenmathieson** [comment]
I've disabled CircleCI on this repo now. No sense in having it complain about build failures if we're no longer using it.

**Comment by Richienb** [comment]
@stephenmathieson 

> I won't necessarily reject the changes. I'm just a bit overwhelmed by the number of changes you're making in a PR entitled "Update cron job".

The deceptive name of the PR is my own fault. I'll fix that.

> ESLint has great defaults on its own. The "recommended" ruleset is usually enough IMO.

Alright. I'll change back.

> Yeah, that's not helpful either.

I'll remove it.

> I have zero interest in partaking in an argument about spaces/tabs. I just don't understand why you opted to change it is all 🤷

I'll revert it to 2 spaces.

**Comment by Richienb** [comment]
@stephenmathieson Ready for re-review!

**Comment by Richienb** [comment]
@stephenmathieson

**Comment by Richienb** [comment]
Whelp, here goes nothing 🤞

**Comment by Richienb** [comment]
@stephenmathieson CircleCi is still being expected via protected branches even though it's no longer reported.

**Comment by stephenmathieson** [comment]
> CircleCi is still being expected via protected branches even though it's no longer reported.

🤦 you're right. I disabled builds in CircleCI but didn't update that here.

**Comment by stephenmathieson** [comment]
I removed branch protection all together. Should be good to go now 👍
